### ISSUE TITLE
Add symfony progress component

### ISSUE THREAD

**Issue by adam-paterson** [meanbee/magedbm#38]
It'd be great to use `Symfony\Component\Console\Helper\ProgressBar` to report `put` and `get` progress in the console.

**Comment by bobbyshaw** [comment]
This would be nice, did you look into this?

**Comment by adam-paterson** [comment]
Not yet, I was thinking of tackling some of the open issues on Friday actually.

**Comment by adam-paterson** [comment]
This will require a big refactor sadly. The version of `Symfony/Console` which n98-magerun uses doesn't have the progress writer. We can alias the version we require but this could introduce a lot of issues.

We'd be better doing this after we remove the magerun dependancy.

**Comment by bobbyshaw** [comment]
Yeah, I just looked again at removing magerun dependency and cba. I guess that's something we will will be pressured to do when looking at the Mage2 version.

**Comment by adam-paterson** [comment]
I'm about half way there removing the magerun dep so I'll continue working on that before starting this.

**Comment by bobbyshaw** [comment]
Oh wow, good effort. See #10 if you haven't already. Nick added an update a while back about using his library for some of the stuff.

**Comment by adam-paterson** [comment]
Sure did, that's the approach I'm taking.
